Src/Support/Google.Apis.Auth/OAuth2/DefaultCredentialProvider.cs

```diff
@@ -228,6 +228,10 @@ private static UserCredential CreateUserCredentialFromParameters(JsonCredentialP
             {
                 throw new InvalidOperationException("JSON data does not represent a valid user credential.");
             }
+            if (credentialParameters.UniverseDomain is not null && credentialParameters.UniverseDomain != GoogleAuthConsts.DefaultUniverseDomain)
+            {
+                throw new InvalidOperationException($"{nameof(UserCredential)} is not supported in other than {GoogleAuthConsts.DefaultUniverseDomain}");
```

Comment:
"not supported in other than" => "not supported in universe domains other than"

Reply:
Done